project-koku/koku
# Cost 2216/2217/2218 - Adding affective cost metric to cost model calculations

This PR add's the ability to store and calculate based from the new effective cost metric.

Testing:
- Check new metrics are visible at http://localhost:8000/api/cost-management/v1/metrics/
- Add ocp source/data
- Create a cost model using effective cost metrics
- Verify calculations are correctly applied to source data
